### PR TITLE
add window check

### PR DESCRIPTION
**Pull request by xiao-lix**

### CODE REVIEW

File: extensions/applicationinsights-analytics-js/src/JavaScriptSDK/Telemetry/PageViewPerformanceManager.ts [hunk 1]
```diff
@@ -92,14 +92,14 @@ export class PageViewPerformanceManager {
...
-        if (this.isPerformanceTimingSupported()) {
+        if (this.isPerformanceTimingSupported() && typeof window === "object") {
             return window.performance.timing;
         }
 
         return null;
     }
     public getPerformanceNavigationTiming(): PerformanceNavigationTiming | null {
-        if (this.isPerformanceNavigationTimingSupported()) {
+        if (this.isPerformanceNavigationTimingSupported() && typeof window === "object") {
```
Comment (hectorhdzg) [inline extensions/applicationinsights-analytics-js/src/JavaScriptSDK/Telemetry/PageViewPerformanceManager.ts]: isPerformanceNavigationTimingSupported already checks for window

File: extensions/applicationinsights-dependencies-js/src/ajax.ts [hunk 2]
```diff
@@ -243,7 +243,7 @@ export class AjaxMonitor implements ITelemetryPlugin, IDependenciesPlugin, IInst
 
     // Fetch Stuff
     protected instrumentFetch(): void {
-        if (!this.supportsFetch() || this._fetchInitialized) {
+        if (!this.supportsFetch() || this._fetchInitialized || !window) {
```
Comment (hectorhdzg) [inline extensions/applicationinsights-dependencies-js/src/ajax.ts]: window check is already in supportsFetch method, add typeof window check in there instead

File: shared/AppInsightsCommon/src/Util.ts [hunk 3]
```diff
@@ -213,7 +213,7 @@ export class Util {
     public static getSessionStorageKeys(): string[] {
         const keys = [];
 
-        if (Util.canUseSessionStorage()) {
+        if (Util.canUseSessionStorage() && typeof window === "object") {
```
Comment (hectorhdzg) [inline shared/AppInsightsCommon/src/Util.ts]: Same here add typeof check inside canUseSessionStorage method

Comment (xiao-lix) [reply shared/AppInsightsCommon/src/Util.ts]: double checked, window is checked in _getVerifiedStorageObject method that is called in canUseSessionStorage method. So no need to check it again.

**Pull request merged**